BUILD.bazel

```diff
@@ -107,3 +121,30 @@ capture_clippy_output(
     build_setting_default = False,
     visibility = ["//visibility:public"],
 )
+
+# This setting may be used to enable builds without the standard library.
+# Currently only no_std + alloc is supported, which can be enabled with setting the value to "alloc".
+# In the future we could add support for additional modes, e.g "core", "alloc,collections".
+string_flag(
+    name = "no_std",
+    build_setting_default = "off",
+    values = [
+        "alloc",
```

Comment:
`no_std` should not be enabling `alloc`! This should be called `no_std_with_alloc`.

Reply:
The flag name itself is `no_std` while the available values currently are `off` and `alloc`, which results in
`--@rules_rust//rust/settings:no_std=off` or `--@rules_rust//rust/settings:no_std=alloc`, which IMO makes it clear when we use alloc.  Do you find `--@rules_rust//rust/settings:no_std=no_std_with_alloc` better?